packages/domain/lib/usecase/get_movie_trending_response_usecase.dart

```diff
@@ -0,0 +1,39 @@
+import '../utils/const.dart';
+import 'package:domain/model/movie_trending_response.dart';
+import '../repository/movie_repository.dart';
+import 'usecase.dart';
+
+class GetMovieTrendingResponseUseCase
```

Comment:
it's better to keep one usecase and pass enum with type of list u want to fetch. no need in duplicating the logic